diff --git a/makefile.vc b/makefile.vc
index e0cf8069e7..8f4f42d81d 100644
--- a/makefile.vc
+++ b/makefile.vc
@@ -60,7 +60,7 @@ TARGET_LIB = staticlib
 
 DEFAULT_TARGETS = 
 
-default:	$(TARGET_LIB) $(PLUGIN_TARGET) apps_dir
+default:	$(TARGET_LIB) $(PLUGIN_TARGET)
 
 # Batch commands to clean main build output are stored in a macro so they can be reused by different
 # targets in an inline temporary batch file.
@@ -227,17 +227,12 @@ install: default
 	if exist $(GDAL_DLL) $(INSTALL) $(GDAL_DLL) $(BINDIR)
 	cd frmts
 	$(MAKE) /f makefile.vc plugins-install
-	cd ..\apps
-	$(MAKE) /f makefile.vc install
 	cd ..
 	$(INSTALL) data\*.* "$(DATADIR)"
 	$(INSTALL) LICENSE.TXT "$(DATADIR)"
 !IFDEF INCLUDE_OGR_FRMTS
 	cd ogr\ogrsf_frmts
 	$(MAKE) /f makefile.vc plugins-install
-!IFDEF HTMLDIR
-	$(MAKE) /f makefile.vc html-install
-!ENDIF
 	cd ..\..
 !ENDIF
 
-- 
2.25.0.windows.1

